src/routes/docs/components/alert.md

```diff
@@ -105,27 +105,27 @@ Use this example to add a border accent to the alert component instead of just a
   import { InfoCircleSolid } from 'flowbite-svelte-icons'
 </script>
 <Alert border>
-  <InfoCircleSolid  class="flex-shrink-0 text-primary-800 dark:text-primary-400 inline w-4 h-4 mr-3" />
+  <InfoCircleSolid  class="text-primary-800 dark:text-primary-400 inline w-4 h-4 mr-3" />
```

Comment:
You've corrected to `slot="icon"` in the first icon example, but it's valid for all alerts with icons: bordered, dismissable, etc

Reply:
Updated.